examples/goldenspike_examples/README.md

```diff
@@ -1,6 +1,6 @@
 This directory contains an example notebook showing a simplified version of the end-to-end functionality of RAIL.
 
-- [goldenspike.ipynb](https://lsstdescrail.readthedocs.io/en/latest/source/other-notebooks.html#goldenspike-an-example-of-an-end-to-end-analysis-using-rail) is a notebook that chains together the functionality of the creation, estimation, and evaluation modules.
+- [goldenspike.ipynb](https://rail-hub.readthedocs.io/projects/rail-notebooks/en/latest/rendered/goldenspike_examples/goldenspike.html) is a notebook that chains together the functionality of the creation, estimation, and evaluation modules.
```

Comment:
Nice catch!

Reply:
all credit to @JaimeRZP 
https://github.com/LSSTDESC/rail/issues/134